tests/Spec/Nri/Ui/Switch.elm

```diff
@@ -0,0 +1,108 @@
+module Spec.Nri.Ui.Switch exposing (..)
+
+import Accessibility.Aria as Aria
+import Accessibility.Role as Role
+import Html.Styled exposing (..)
+import Nri.Ui.Switch.V2 as Switch
+import ProgramTest exposing (..)
+import Test exposing (..)
+import Test.Html.Selector as Selector
+
+
+spec : Test
+spec =
+    describe "Nri.Ui.Switch.V2"
+        [ describe "'switch' role" hasCorrectRole
+        , describe "has aria-disabled=true when disabled" hasCorrectAriaDisabled
```

Comment:
So these tests don't actually test state at all, which is fine with me, but I think if we're not testing state, we don't need to use ProgramTest -- might as well keep things simpler if we can, you know?